### DISCUSSION TITLE
Startup and Backlight Issues on Frsky X-Lite S/Pro

### DISCUSSION THREAD

**Discussion by BradyWags** [EdgeTX/edgetx#3578] [Q&A]
I just changed from OpenTX to EdgeTX 2.8.3 but I am having some issues with turning the transmitter on and the backlight.
I have an Frsky X-lite S/Pro
I changed as I just got an ELRS system for my drone.

When I turn on the transmitter, it displays a blank screen for about 4s and then it starts the loading bars to turn on. Sometimes instead of a blank screen it displays the person going to sleep thing that displays when turning off and sometimes it displays a bunch of black spots. I changed the power on delay to 0s to make it faster but it still takes a long time.
Also, the throttle not idle warning pops up every single time no matter what.

The backlight has never once turned on since I upgraded. I have tried all the different backlight modes and have the brightness set to 100. It just never lights up now.

I have re-flashed multiple times and even tried a few older versions with not luck. Everything worked just fine on OpenTX so the radio should be fine.

**Comment by pfeerick** [comment]
Can I just double check that you are running the xlites firmware, not the xlite firmware? i.e. in SYS -> VERSION it should show something like this - with FW `edgetx-xlites` being the important bit.

![image](https://github.com/EdgeTX/edgetx/assets/5500713/4ebb12a7-1fd8-44b2-9d6f-536cad83da6a)

I'm not sure why it's showing a blank screen for a period... it should basically show the dots counting up for the time period set for the power on delay time, then the splash screen and any warnings.

**Reply by BradyWags** [reply]
I do for sure have the xlites firmware not the xlite. 

The 4 dots do not show up until I have held down the power button for quite a while and then it pops up. It feels like its doing the power on pause twice almost. I do not have a splash screen set.

![IMG_3462](https://github.com/EdgeTX/edgetx/assets/86738587/8969b28c-eeee-4637-ab9b-510c886a2014)
